targets/TARGET_STM/TARGET_STM32F7/TARGET_STM32F756xG/device/TOOLCHAIN_IAR/stm32f756xg.icf

```diff
@@ -1,27 +1,59 @@
-if (!isdefinedsymbol(MBED_APP_START)) { define symbol MBED_APP_START = 0x08000000; }
-if (!isdefinedsymbol(MBED_APP_SIZE)) { define symbol MBED_APP_SIZE = 0x100000; }
+/* Linker script to configure memory regions.
+ ******************************************************************************
+ * @attention
+ *
+ * Copyright (c) 2016-2019 STMicroelectronics.
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
```

Comment:
Can you add SPDX identifier to files with he license like this one? Otherwise we need to create own rules to get the license like this (as it only references the license similar to SPDX)